guides/advanced_guides/search_parameters.md

```diff
@@ -139,7 +139,7 @@ You will get the following response with the **cropped version in the \_formatte
     "id": "50393",
     "title": "Kung Fu Panda Holiday",
     "poster": "https://image.tmdb.org/t/p/w1280/gp18R42TbSUlw9VnXFqyecm52lq.jpg",
-    "overview": "his father hang decorations, cook together, and serve noodle soup to the villagers. But this year Shifu informs Po that as Dragon Warrior, it is his duty to host the formal Winter Feast at the Jade",
+    "overview": "this year Shifu info",
```

Comment:
I tried the command and this was my result
```suggestion
    "overview": "this year Shifu informs",
```

Reply:
Thank you!